ui/src/views/Settings.vue

```diff
@@ -264,4 +609,13 @@ export default {
 
 <style scoped lang="scss">
 @import "../styles/carbon-utils";
-</style>
\ No newline at end of file
+.mg-left {
+  margin-left: 2em;
```

Comment:
```suggestion
  margin-left: 2rem;
```
Let's always use `rem`